janusgraph-core/src/main/java/org/janusgraph/core/JanusGraphElement.java

```diff
@@ -60,18 +60,7 @@ public interface JanusGraphElement extends Element, Idfiable, Removable {
      * @see #hasId
      */
     @Override
-    default Object id() {
-        return longId();
-    }
-
-    /**
-     * Unique identifier for this entity. This id can be temporarily assigned and might change.
-     * Use {@link #id()} for the permanent id.
-     *
-     * @return Unique long id
-     */
-    @Override
-    long longId();
```

Comment:
Do we want to deprecated this function before removing?

Reply:
Good idea!